app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/package.scala

```diff
@@ -309,5 +309,33 @@ package object controllers extends Logging {
           submittedAnswer => Seq(Yes, No).contains(submittedAnswer.check)
         )
     )
+
+    val contactPhoneCheckForm = Form[ContactPhoneCheck](
```

Comment:
could we look at moving these forms from this package file into a separate 'forms' directory? just seems a bit odd compared to our other FE services

Reply:
yes of course....probably a good idea. We can make a ticket for that.